.github/ISSUE_TEMPLATE/courses.yaml

```diff
@@ -0,0 +1,83 @@
+name: \[COURSES\] Typo report or new content request
+description: Report typos, bugs, out-of-date content, broken links, etc. or make a request for new content
+labels: ["content 📄", "course 🎓", "needs triage 🤔"]
+assignees:
+  - jhwatrous
+  - livlanes
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for reporting or requesting new content!  Please also use the search to see if there are any other relevant issues or pull requests.
+  - type: markdown
+    attributes:
+      value: |
+        ⚠️ Note that this issue template is only for course content; verify that /learning/ is in the URL.
+  - type: markdown
+    attributes:
+      value: |
+        * If your issue concerns tutorials (with /tutorials/ in the URL), use the [Tutorials issue template](https://github.com/Qiskit/documentation/issues/new?template=tutorials.yaml) instead.
+  - type: markdown
+    attributes:
+      value: |
+        * If your issue concerns guides or additional resources, use the [Guides issue template](https://github.com/Qiskit/documentation/issues/new?template=guides.yml) instead.
+  - type: markdown
+    attributes:
+      value: |
+        * If your issue concerns Qiskit API content, open an issue in the [Qiskit repository](https://github.com/Qiskit/qiskit/issues/new/choose).
+  - type: markdown
+    attributes:
+      value: |
+        * If your issue concerns Qiskit Runtime API content, open an issue in the [Qiskit IBM Runtime repository](https://github.com/Qiskit/qiskit-ibm-runtime/issues/new/choose).
+  - type: markdown
+    attributes:
+      value: |
+        * If your issue concerns a Qiskit Ecosystem project, open an issue in the relevant [Ecosystem project repository](https://qiskit.github.io/ecosystem/).
```

Comment:
This URL is now https://www.ibm.com/quantum/ecosystem and should be updated in every template